packages/utilities/gel-foundations/src/scripts.js

```diff
@@ -1,15 +1,28 @@
-export latin from './scripts/latin-and-cyrillic';
-export cyrillic from './scripts/latin-and-cyrillic';
-export latinDiacritics from './scripts/latin-with-diacritics';
 export arabic from './scripts/arabic';
+
 export arabicPashto from './scripts/arabic-pashto';
+
 export bengali from './scripts/bengali';
+
 export burmese from './scripts/burmese';
+
+export devanagariAndGurmukhi from './scripts/devanagari-and-gurmukhi';
 export hindi from './scripts/devanagari-and-gurmukhi';
 export nepali from './scripts/devanagari-and-gurmukhi';
+
+export cyrillic from './scripts/latin-and-cyrillic';
+export cyrillicAndLatin from './scripts/latin-and-cyrillic';
+export latin from './scripts/latin-and-cyrillic';
+
+export latinDiacritics from './scripts/latin-with-diacritics';
+
+export chinese from './scripts/no-ascenders-or-descenders';
```

Comment:
At this point, should we still export `chinese`, `ethiopic` and `korean`?

Reply:
@DenisHdz I'm going to remove the services names as exports from this `gel-foundations` in a future PR. But for now, I'm ensuring we are exporting the script values.

Reply:
It makes sense to me, thanks :)